README.md

```diff
@@ -1 +1,76 @@
+
+**Dataset name** - it's may be public data e.g. MSCOCO or Open images, or your internal dataset.
+
+**Preview image** - we will run a model on the image and will use its output for visualization. This image will be both on the [main page](https://modelplace.ai/) and on the [model page](https://modelplace.ai/models/2)
+
+**Licence** - licence type e.g. Apache 2.0, MIT, Proprietary and so on
+**Number of classes** 
+**Author** - your name or company name
+**Metrics** - accuracy of your model
+
+### Optional
+**Dataset link** - link to dataset that you use for training and testing of your model
+**Source link** - link to your project page or profile page or company page and so on
+**System requirements** - what system setup( e.g. number of CPU GPU RAM) we should use to serve your model.
+
+**Logo** - we will add it to the preview.   
+_Note: This should be svg(png is possible) image with circle shape and size 128x128 pixels at least._
+
+**Avatar image** - you can add your own logo(by default we use our).   
+_Note: This should be svg(png is possible) image with circle shape and size 128x128 pixels at least._
+
+## Evironment
+- Install venv  
+```python3.7 -m pip install virtualenv```
+- Create an empty virtual environment for python3.7  
+```python3.7 -m  virtualenv  venv```
+- Activate it  
+```source venv/bin/activate```
+- Install pytest and wheel  
+```python3.7 -m pip install pytest wheel```
+- Install git and git-lfs  
+```sudo apt install git git-lfs```
+- Clone the repo 
+```git clone https://github.com/opencv-ai/modelplace.git```
+
+## Example
+You can run [an example](pytorch_fastercnn) of a package
+- Go to package folder pytorch_fastercnn
+```cd pytorch_fastercnn```
+- Install the package  
+```python3.7 setup.py install develop```
+- Run tests  
+```pytest pytorch_fastercnn```
+
+Example of model usage you can find [here](https://modelplace.ai/models/2).
+
+## Package
+We use an abstraction to unify model serving. You should create [a wheel package](https://packaging.python.org/tutorials/packaging-projects/#creating-the-package-files) with our interfaces. You can find example [here](pytorch_fastercnn).
+
+To create a package follow the steps:
+- Copy the template folder
+- Extend model.py as it's done in [an example](pytorch_fastercnn/pytorch_fastercnn/model.py)
+- Update [the setup.py](template/setup.py)
+- Copy your checkpoints to [the template/checkpoints](template/template/checkpoints) folder
+- Rename "template" to your package name in all locations, including the folder name, e.g. template -> pytorch_fastercnn
+- Install the package locally  
+```python3.7 setup.py install develop```
+- Update tests in [test_model.py](template/tests/test_model.py)
+- Run tests  
+```python3.7 -m pytest tests```
+- Make sure that the tests are working correctly and passed
+- Zip the folder with the completed template
+- Send us this zip and information about package
```

Comment:
Don't we want to specify the format of information about package? Probably we should provide something like json template for this data?

Reply:
I think we can put the description.json file in template and pytorch_fastercnn folders. We don't need to have all the internal information there. Only those which we ask in `Publishing information`

Reply:
It is better to have it in an email to immediately discuss it with the developer.